drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ActionRetractFactPlugin.java

```diff
@@ -92,8 +94,13 @@ public Boolean generateColumn() {
         if (isNewColumn()) {
             presenter.appendColumn(editingCol());
         } else {
-            presenter.updateColumn(originalCol(),
-                                   editingCol());
+            try {
+                presenter.updateColumn(originalCol(),
+                                       editingCol());
+            } catch (VetoException veto) {
+                wizard.showGenericVetoError();
```

Comment:
I saw few `try catch` blocks that just `swallow` the exception, shouldn't they show some message as here? I don't know, just a question.

Reply:
I _know_ as the developer that these scenarios should not happen at the moment (as there are no cases where we'd throw a ```VetoException``` in those scenarios). However I have made them report a generic "veto" message instead - for future proofing.

Reply:
Thanks for explanation.